node_modules/oae-authentication/tests/test-auth-local.js

```diff
@@ -1173,55 +1205,75 @@ describe('Authentication', function() {
+                                    assert.ok(err);
+                                    assert.equal(err.code, 400);
+                                    assert.ok(!user);
 
-                                // Verify the context was not authenticated
-                                RestAPI.User.getMe(testGlobalUserRestContext, function(err, me) {
-                                    assert.ok(!err);
-                                    assert.ok(me.anon);
+                                    // Ensure the global admin still cannot be authenticated
+                                    var testGlobalUserRestContext = TestsUtil.createGlobalRestContext();
 
-                                    // Sanity check the user can be created through the REST endpoints
-                                    RestAPI.User.createGlobalAdminUser(globalAdminRestContext, username, username, username, null, function(err, user) {
-                                        assert.ok(!err);
-                                        assert.ok(user);
+                                    // Ensure that the credentials do not authenticate a value global administrator
+                                    RestAPI.Authentication.login(testGlobalUserRestContext, username, username, function(err) {
+                                        assert.ok(err);
+                                        assert.equal(err.code, 401);
 
-                                        // Ensure when we try and create one with the same loginId, we get a 400 error
-                                        RestAPI.User.createGlobalAdminUser(globalAdminRestContext, username, username, username, null, function(err, secondUser) {
-                                            assert.ok(err);
-                                            assert.equal(err.code, 400);
+                                        // Verify the context was not authenticated
+                                        RestAPI.User.getMe(testGlobalUserRestContext, function(err, me) {
+                                            assert.ok(!err);
+                                            assert.ok(me.anon);
 
-                                            return callback();
+                                            // Sanity check the user can be created through the REST endpoints
+                                            RestAPI.User.createGlobalAdminUser(globalAdminRestContext, username, username, username, email, {}, function(err, user) {
+                                                assert.ok(!err);
+                                                assert.ok(user);
+
+                                                // Ensure when we try and create one with the same loginId, we get a 400 error
+                                                RestAPI.User.createGlobalAdminUser(globalAdminRestContext, username, username, username, email, {}, function(err, secondUser) {
```

Comment:
In principle, it's more desirable if one person can have multiple accounts in different tenants, especially in order to satisfy some of the Fold / STEM incubator use cases we currently have. Having said that, if we want a way in which we can share something with a person through an email address, I don't see a way around the restriction as we would have no idea which account to match that with.